src/SourceGenerators/Uno.UI.SourceGenerators/DependencyObject/DependencyObjectGenerator.cs

```diff
@@ -175,18 +175,15 @@ private void ProcessType(INamedTypeSymbol typeSymbol)
 						}
 					};
 
-					var canBeTpProvider = !typeSymbol.Interfaces.Any(x => x.Name == "INotTemplatedParentProvider");
```

Comment:
This `INotTemplatedParentProvider` interface is no longer needed? Can it be removed?

Reply:
I see it's already removed. The quick CTRL+F in browser didn't point it out because file name is different and diff was collapsed.